code-challenges.test.js

```diff
@@ -32,28 +57,106 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+describe('divisibleByThree', () => {
+	it('should determine if property number is divisible by 3', () => {
+		expect(divisibleByThree(object1)).toEqual('15 is divisible by three')
+		expect(divisibleByThree(object2)).toEqual('0 is divisible by three')
+		expect(divisibleByThree(object3)).toEqual('-7 is not divisible by three')
+	})
+})
+
 // b) Create the function that makes the test pass.
 
+// create a function that accepts an object as an argument
+// destructure number from object
+// if number is divisible by 3 then return 'number is divisible by three'
```

Comment:
Awesome job! One of the things that will make your pseudocode stand out more is using vocab like modulo and strict equality operator.